Documentation/Track2TestFramework.md

```diff
@@ -0,0 +1,166 @@
+# Accquring TestFramework
+
+To start using test framework import `sdk\core\Azure.Core\tests\TestFramework.props` into test `.csproj`:
+
+``` xml
+<Project Sdk="Microsoft.NET.Sdk">
+
+...
+  <Import Project="..\..\..\core\Azure.Core\tests\TestFramework.props" />
+...
+
+</Project>
+
+```
+
+# Sync-async tests
+
+The test framework provides an ability to write test using async client methods and automatically run them using sync overloads. To write sync-async client tests inherit from `ClientTestBase` class and use `InstrumentClient` method call to wrap client into a proxy class that would automatically forward async calls to their sync overloads.
+
```

Comment:
It'd be probably be good to call out
- `TestClassName(true)` in the test runner corresponds to the async calls and `TestClassName(false)` corresponds to the sync calls
- This will create two test recordings and the async calls will have the suffix `Async.json`
- You can disable the sync-forwarding for an individual test by applying the `[AsyncOnly]` attribute